src/components/OrganizationHeader/OrganizationHeader.tsx

```diff
@@ -92,10 +92,15 @@ const useStyles = makeStyles(theme => ({
 }));
 
 interface Props {
+  name?: string;
   buttonText: string;
   onButtonClick: () => void;
 }
-export const OrganizationHeader = ({ buttonText, onButtonClick }: Props) => {
+export const OrganizationHeader = ({
+  name,
+  buttonText,
+  onButtonClick,
+}: Props) => {
```

Comment:
in this case, `useSelectedCircle` ought to be passed in from parent as well (perhaps for another PR)